lib/puppet/pops/serialization/from_data_converter.rb

```diff
@@ -160,7 +160,7 @@ def pcore_type_hash_to_value(pcore_type, value)
       elsif value.is_a?(String)
         build(pcore_type.create(value))
       else
-        raise SerializationError, _('Cannot create a %{type_name} from a %{arg_class') %
```

Comment:
Please add a unit test for this. I assume there isn't one, otherwise we should have seen a failure and appveyor/travis would have failed before merging?